hw/bsp/sltb009a/board.mk

```diff
@@ -0,0 +1,45 @@
+CFLAGS += \
```

Comment:
please add the following line, that will help to fix the CI. I have tried may the build to fetch needed submodules (if not already) without user have to do it manually or init it all.

```
DEPS_SUBMODULES += hw/mcu/silabs/cmsis-dfp-efm32gg12b
```

Reply:
I did add that, it's a bit lower down.

Reply:
ah right, it is probably need to have the #703 merged first (and/or) update/rebase aftewrads.